gfx.cpp

```diff
@@ -96,13 +96,13 @@ class GfxInternal
     ID3D12GraphicsCommandList *command_list_ = nullptr;
     ID3D12GraphicsCommandList4 *dxr_command_list_ = nullptr;
     ID3D12GraphicsCommandList6 *mesh_command_list_ = nullptr;
-    ID3D12CommandAllocator **command_allocators_ = nullptr;
+    ID3D12CommandAllocator *command_allocators_[kGfxConstant_MaxBackBufferCount] = {};
```

Comment:
Why this change? These can't be statically-allocated arrays as another way of using `gfx` is by creating an "interop" context in which the user can supply an arbitrary value for `max_frames_in_flight`:
https://github.com/gboisse/gfx/blob/d617b252a68ea27116a5a26027530c22d45bc9e6/gfx.h#L450

In such a case, `gfx` doesn't own the number of backbuffers, etc. but has to align to the application being "interop'd" with and we can't rely on the `kGfxConstant_MaxBackBufferCount` constant.